content/home/home-content.md

```diff
@@ -4,7 +4,7 @@
 
 EPA is overseen by two congressional committees, the House Energy and Commerce Committee (E&C) and the Senate Environmental and Public Works Committee (EPW). EDGI took a look at environmental enforcement in committee members' home districts. Here's where those reps come from:
 
-![Map of representatives on EPA oversight committees, covered by EEW Congressional Report Cards project](./eew-map.png)
+[![Map of representatives on EPA oversight committees, covered by EEW Congressional Report Cards project](eew-map.png)](https://edgi-govdata-archiving.github.io/EEW-Website/reports "Map of representatives on EPA oversight committees, covered by EEW Congressional Report Cards project")
```

Comment:
Link target should be `/reports`, which is a relative link. We'll hit the error outlined in https://github.com/edgi-govdata-archiving/EEW-Website/issues/163 but I will put a note on that issue to check this out at the same time (when we go live)